### ISSUE TITLE
Can't add links

### ISSUE THREAD

**Issue by bardullah** [Haroenv/floating.js#18]
I've tried to add a link as  content: '< a  href=" http://www.facebook.com ">FB </ a > and it shows it as a link (blue & underlined) but it's not clickable. When I try to click nothing happens.

**Comment by Haroenv** [comment]
Yes, the css has `pointer-events: none;`, let me check if I made it able to be disabled

**Comment by Haroenv** [comment]
Do you want to make a PR to add an option “clickable” (defaulting to false) which disables that part of the css?
